multi_agent_coverage_planner/Point2D[] data
